assets/api-swagger/swagger.json

```diff
@@ -1812,7 +1872,7 @@
           },
           "store": {
             "type": "string",
-            "description": "Code representing the store the order was placed in."
+            "description": "Code representing the store the order was placed in.\nmax length: 32"
```

Comment:
Should we be using the `maxLength` to specify this?

Reply:
I followed the examples in other sections of the file, like in l. 1806

Reply:
Fair we seem to do both.